src/Interpreter/Executors/Bash/Executor.php

```diff
@@ -95,4 +97,12 @@ private function buildCommand(string $script, array $args, ?string $cwd): array
             ...array_values($args),
         ];
     }
+
+    /**
+     * @return array<string,string>
+     */
+    private function buildEnv(): array
+    {
+        return getenv();
```

Comment:
it is apparently, since 8.0, but try
```suggestion
        return getenv(null, true);
```

Reply:
maybe it's default null is just not supplied and confused

Reply:
I tried that locally but phpstan wasn't having it (even latest phpstan version).

Reply:
sorry joke incoming
```suggestion
        $env = [];
        foreach (array_keys(getenv()) as $key) {
            $value = getenv(key, true);
            if ($value !== null) {
                 $env[$key] = $value;
            }
        }
        return $env;
```

at least no http request variables in cli though so nvm